src/component/person.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable max-len */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './person.scss';
+
+const Married = (isMarried, sex, partnerName) => {
+  let message = '';
+
+  if (isMarried === false) {
+    message = 'I am not married';
+  } else if (isMarried === true) {
+    if (sex === 'f') {
+      message = `My husband's name is ${partnerName}`;
+    } else {
+      message = `My wife's name is ${partnerName}`;
+    }
+  }
+
+  return message;
+};
+
+export const Person = ({ person }) => {
+  return (
+    <section className="Person">
```

Comment:
Fix class naming, use small first letters
```suggestion
    <section className="person">
```